config/initializers/constants.rb

```diff
@@ -20,6 +20,7 @@
 BASE_ASSUMPTIONS_URL = EXTERNAL_SITE_URL + "/base_assumptions"
 POLICIES_URL = EXTERNAL_SITE_URL + "/policies"
 MEMBERSHIP_URL = EXTERNAL_SITE_URL + "/membership"
+MEMBERSHIP_BENEFITS_RESPONSIBILITIES_URL = MEMBERSHIP_URL + "/#membership-benefits-and-responsibilities"
```

Comment:
You are right that the server needs to be restarted to pick up this new constant. This will happen automatically during the deploy -- the code will be pushed and the server then restarted (by Heroku). 👍🏼